numpy/lib/function_base.py

```diff
@@ -4819,6 +4819,13 @@ def trapz(y, x=None, dx=1.0, axis=-1):
     >>> np.trapz([1,2,3], dx=2)
     8.0
 
+    Numerical integration of a simple function on an interval:
```

Comment:
```suggestion
    Estimate the integral :math:`\int)^1 x^2 = 1/3`:
```
(I might also suggest passing `num=` to linspace, I don't like the implicit number of steps a lot.  I am not 100% sure the math there works, the CI will tell us)